compiler/src/main/kotlin/asmble/cli/ScriptCommand.kt

```diff
@@ -86,10 +87,21 @@ abstract class ScriptCommand<T> : Command<T>() {
                 throw Exception("Failed loading $inFile - ${e.message}", e)
             }
         }
+
+        val mem = args.memoryBuilder
```

Comment:
We have `Mem` interface defined in Asmble and `mem` as a local there. Maybe change name to smth else.